synapse/lib/cell.py

```diff
@@ -2712,54 +2685,13 @@ async def iterNewBackupArchive(self, user, name=None, remove=False):
                     logger.debug(f'Removed {path}')
                 raise
 
-            await self.boss.promote('backup:stream', user=user, info={'name': name})
```

Comment:
No it was moved into the helper function `_streamBackupArchive`

Reply:
Actually looking closely it was removed because _streamBackupArchive is invoked from both `iterBackupArchive` and `iterNewBackupArchive` so it became redundant.